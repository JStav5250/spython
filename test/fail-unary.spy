str a = "2"

a = -3
