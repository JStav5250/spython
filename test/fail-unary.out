TypeError: expression of type int cannot be assigned to variable 'a' with explicit type str!!
